From c7fb85ca649e3dd9f2efc50ac03000a5b42b0127 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Mon, 27 Nov 2017 23:44:47 +0000
Subject: [PATCH] Disable use of the Clear Containers runtime

Currently there is an issue using p9fs with overlays on the 4.14 Linux
kernel. This causes poor behavior in Clear Containers so for now only
use runc in Docker until the problem is resolved.
---
 daemon/config/cc-daemon.go | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/daemon/config/cc-daemon.go b/daemon/config/cc-daemon.go
index a297eeb..5884bec 100644
--- a/daemon/config/cc-daemon.go
+++ b/daemon/config/cc-daemon.go
@@ -3,7 +3,6 @@ package config
 import (
 	"github.com/Sirupsen/logrus"
 	"github.com/docker/docker/api/types"
-	"os"
 	"os/exec"
 )
 
@@ -15,9 +14,6 @@ const (
 )
 
 func hasKVM() bool {
-	if _, err := os.Stat("/dev/kvm"); err == nil {
-		return true
-	}
 	return false
 }
 
-- 
2.15.0

